### PR TITLE
Enable PullPreview action (pullpreview.com)

### PR DESCRIPTION
**Pull request by crohr**
This adds a docker-compose for use with the [PullPreview action](https://pullpreview.com), complete with HTTPS support from Let's Encrypt.

To avoid repetition you could switch to a base docker-compose.yml file, and then have a docker-compose.development.yml and docker-compose.pullpreview.yml, each with just the extensions required. See https://docs.docker.com/compose/extends/ for details.

For this to work in your repository, you will have to add 2 secrets into the "Secrets" section of the GitHub repo:

* `AWS_ACCESS_KEY_ID`
* `AWS_SECRET_ACCESS_KEY`

I would recommend that you create at least a separate IAM user for this (see https://github.com/pullpreview/action/wiki/Recommended-AWS-Configuration).

And to test on a PR you'll also need to create the `pullpreview` label, and apply it to the PR. Currently it will always auto-deploy the `master` branch. If you don't want this, remove the `always_on` option in the workflow file.

Let me know if I can help with anything else! The working PR against my repo is at https://github.com/pullpreview/chaskiq/pull/1.

### CODE REVIEW

File: .github/workflows/pullpreview.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v2
+    - uses: pullpreview/action@v3
+      with:
+        admins: crohr,michelson
```
Comment (michelson) [inline .github/workflows/pullpreview.yml]: it `crohr` user necessary here ?

File: db/seeds.rb [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
+  )
 
-Doorkeeper::Application.create(
-   name: "authapp", 
-   #redirect_uri: "#{domain}/callback"
-)
+  Doorkeeper::Application.create(
+     name: "authapp", 
+     #redirect_uri: "#{domain}/callback"
+  )
```
Comment (michelson) [inline db/seeds.rb]:
```suggestion
```

Comment (michelson) [reply db/seeds.rb]: this is not necessary anymore

**Review by michelson — CHANGES_REQUESTED** [review]
Thanks for the PR Cyril, I've leaved a couple of comments.

I've my AWS / GH environment prepared to merge try this out

best

**Comment by crohr** [comment]
@michelson thanks, I addressed your comments.
When you're ready, merge the PR and you should get an env for `master`.

Then you can create the `pullpreview` label and assign it to any new PR so that it will launch a preview environment for that PR. If you want to use pullpreview for existing PRs, make sure you rebase the PR from `master` so that you get the pullpreview.yml workflow file.

**Pull request closed without merging**